packages/connect-react/src/hooks/customization-context.tsx

```
@@ -0,0 +1,233 @@
+import { Field } from "../components/Field"
+import { Label } from "../components/Label"
+import { OptionalFieldButton } from "../components/OptionalFieldButton"
+
+export const defaultComponents = {
+  Description,
+  Errors,
+  Label,
+  OptionalFieldButton,
+}
+
+export type ReactSelectComponents = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ReactSelectComponents'. Consider to delete it or rename it.

'ReactSelectComponents' is defined here:


(lint/suspicious/noRedeclare)